### PR TITLE
The first SummerSchool

### PR DESCRIPTION
**Pull request by isabelheisters**
So this is much of text on this website. Maybe we should delete  something of that.
And is it okay in present tense or should I write this one in past tense too?
Signed-off-by: Isabel Heisters i.heisters@fz-juelich.de

### CODE REVIEW

File: _events_past/2014-06-12-1st-jlesc-summer-school.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Modern computing systems have become increasingly complex and large scale. 
+Understanding their performance has thus become dramatically more difficult than it has ever been.
+The aim of this summer school is to provide attendees a presentation of modern tools and techniques 
+which allow to study the performances of large HPC systems. Four series of lectures (including hands on) 
+on trace/workload analysis/visualization and system simulation/emulation will be given by domain experts. 
+Although classical API (s.a MPI, CHARM++, OmpSs) will be used to illustrate techniques, the fundamental 
+problems remain identical and interesting to any student working with HPC systems and less classical APIs 
+or applications. 
+
+Thursday, June 12th 2014, 8:30AM-12PM:<br/> HPC Application Tracing and Analysis (J. Labarta)
```
Comment (torbjoernk) [inline _events_past/2014-06-12-1st-jlesc-summer-school.md]: Don't use HTML `<br />` in Markdown. For intra-paragraph line breaks, Markdown has two spaces at the end of the line.

**Comment by pancetta** [comment]
Present tense is fine here IMHO.

**Comment by pancetta** [comment]
I agree, this is a lot of text. @torbjoernk how about having a subpage with the schedule?

**Comment by torbjoernk** [comment]
I just thought about a sub-page similar to the one for the Barcelona workshop as well.

@isabelheisters: should I prepare the scaffold for that or do you want to try it on your own based on the schedule of the Barcelona workshop?

**Comment by pancetta** [comment]
Well, don't put in too much effort. This is an old event.. just copy the
plain schedule to a sub-page and that's it. The BSC event already
demonstrates how this _could_ look like, if you really want.

**Comment by isabelheisters** [comment]
I could try to make a subpage for the schedule. The Barcelona subpage is made with html. @torbjoernk: Can I do this with markdown too?

**Comment by torbjoernk** [comment]
@isabelheisters When you don't want the fancy collapsing containers, plain Markdown works fine as well. You could still mix in some HTML. The Barcelona's schedule file is only a HTML file so that it doesn't go through the Markdown parser (and for better syntax highlighting in editors).

**Comment by isabelheisters** [comment]
Okay I will try =)

**Comment by pancetta** [comment]
@isabelheisters: Very nice!!

**Pull request merged**